src/sources/http_client/scrape.rs

```diff
@@ -33,10 +33,10 @@ use vector_core::{
     event::Event,
 };
 
-/// Configuration for the `http_scrape` source.
-#[configurable_component(source("http_scrape"))]
+/// Configuration for the `http_client` source.
+#[configurable_component(source("http_client"))]
 #[derive(Clone, Debug)]
-pub struct HttpScrapeConfig {
+pub struct HttpClientConfig {
     /// Endpoint to scrape events from. The full path must be specified.
```

Comment:
Wonder if we should rephrase this as well to something like "Endpoint to gather events from. <...>" ?
Same goes for the cue file.

Reply:
👍 gather/collect is a good change there